docs/research/usace-campgrounds-master-list.md

```
@@ -0,0 +1,61 @@
+# USACE Huntington District - Master Campground Recreation.gov IDs
+
+Research completed: 2025-12-15
+
+## Verified Recreation.gov IDs
+
+### Sutton Lake (25 min from shop)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)